### DISCUSSION TITLE
Server AWS Region Singapore, Zone A (ap-southeast-1a)

### DISCUSSION THREAD

**Discussion by taufiqsetiawan-mboton** [awsugid/forum#30] [Q&A]
Hallo AWS Forum Indonesia

Saya ingin bertanya? terkait dengan permasalahan pada akun wordpress saya yang mengalami gangguan yaitu tidak bisa masuk Connect menggunakan SSH client berbasis browser kami (blank Hitam) Dan website saya mengalami A timeout occurred Error code 524 dan 520.

RATA-RATA UTILISASI CPU PER 5 MENIT
1,25%
KAPASITAS CPU BURST YANG TERSISA (PERSENTASE)
63.56%

Connect menggunakan SSH client saya mendapatkan pesan seperti dibawah ini.

1. The remote server has closed the connection because your instance is taking too long to respond. Try again or contact customer support. UPSTREAM_TIMEOUT [514]
2. Your instance encountered an error and has closed the connection. Try again or contact customer support. UPSTREAM_ERROR [515]

Ini solusi bagaimana ya? Dan saya harus bagaimana agar website aktif seperti sebelumnya

Terima kasih
Taufiq Setiawan

![Github](https://github.com/awsugid/forum/assets/122861473/abfcda8e-9b6b-49f1-ae5c-f83504a97229)

**Comment by rioastamal** [comment]
@taufiqsetiawan-mboton Apa masalahnya sudah teratasi? Apakah SSH dari terminal biasa masih bisa dilakukan? Kemungkinan CPU dari instance sedang spike atau RAM udah penuh sehingga terjadi kernel melakukan OOM (Out of Memory) killer.

Saran saya tambahkan swap pada instance Lightsail sebesar 1GB atau 2GB. Kurang lebih langkahnya:

```
sudo dd if=/dev/zero of=/var/swapfile bs=1M count=1024
sudo chmod 066 /var/swapfile
sudo mkswap /var/swapfile
sudo swapon /var/swapfile
```

Jika ingin permanent tambahkan entry di `/etc/fstab`, cek referensi di internet.

**Comment by taufiqsetiawan-mboton** [comment]
@rioastamal Masalah sudah teratasi dengan baik? saya sudah melakukan perubahan dengan baik, sehingga saat ini instance tidak ada troble lagi. 

Terima Kasih
